pkg/prometheus/promcfg_test.go

```diff
@@ -534,16 +575,17 @@ func TestPodTargetLabels(t *testing.T) {
 				},
 			},
 		},
-		map[string]*monitoringv1.ServiceMonitor{
-			"testservicemonitor1": &monitoringv1.ServiceMonitor{
+		nil,
+		map[string]*monitoringv1.PodMonitor{
```

Comment:
We still want the test that was here previously. Could you extract this into a separate test?

Reply:
Sure